TypeCobol.LanguageServer/TypeCobolServerHost.cs

```diff
@@ -210,6 +215,9 @@ static int Main(string[] args)
                 { "dcs|disablecopysuffixing", "Deactictivate Euro Information suffixing", v => UseEuroInformationLegacyReplacingSyntax = false },
                 { "sc|syntaxcolor",  "Syntax Coloring Support.", _ => UseSyntaxColoring = true},
                 { "ol|outlineRefresh",  "Outline Support.", _ => UseOutlineRefresh = true},
+                { "cfg=",  "{dot output mode} Control Flow Graph support and Dot Output mode: No/0, AsFile/1 or AsContent/2.",
+                    (String m) => {TypeCobolCustomLanguageServer.UseCfgMode ucm = TypeCobolCustomLanguageServer.UseCfgMode.No;
+                        Enum.TryParse(m, false, out ucm); UseCfg = ucm; }  },
```

Comment:
no need to specify the `ignoreCase` to `false`, it is the default behavior.

Reply:
Done in 
c771e78